src/theme.jsx

```diff
@@ -94,13 +94,20 @@ const theme = createMuiTheme({
         width: 41,
         height: 41,
         color: '#000',
+        padding: 0,
       },
     },
     MuiDrawer: {
       paperAnchorDockedLeft: {
         borderRight: 'none',
       },
     },
+    MuiListItem: {
+      root: {
+        paddingTop: '12px',
+        paddingBottom: '12px',
+      },
+    },
```

Comment:
Why are we adding `MuiListItem`?

Reply:
The padding is different in the Sidebar, Material v2 has 11px while the v1 has 12px.

Reply:
Ah I see. It should still be okay to remove it. The UX still look nice :)